usabilla-react-native/README.md

```diff
@@ -44,6 +44,12 @@ To use this Bridge, please make sure you are using XCode 9.1 or above.
 - Submit the results of the form.
 - Pre-fill a Passive Feedback form with a custom screenshot.
 
+Usabilla for React Native allows you to attach screenshots to a form before sending it by calling:
```

Comment:
I would use "a screenshot" instead of "screenshots" as we can have only one per form